src/hooks/useAppState/reducer.js

```diff
@@ -59,6 +59,10 @@ const reducer = (state, { type, data }) => {
           return {
             ...list,
             items: list.items.filter(({ key }) => key !== data.key),
+            deletedItems: [
+              ...list.deletedItems ?? [],
+              ...list.items.filter(({ key }) => key === data.key),
```

Comment:
Storing a list of deleted items, we'll need this to be able to undo deletion in the future